src/dlgTriggerEditor.cpp

```diff
@@ -182,18 +184,32 @@ dlgTriggerEditor::dlgTriggerEditor( Host * pH )
     QSizePolicy sizePolicy5(QSizePolicy::Expanding, QSizePolicy::Expanding);
     mpSourceEditorArea->setSizePolicy( sizePolicy5 );
     pVB1->addWidget( mpSourceEditorArea );
-    mpSourceEditor = mpSourceEditorArea->editor;
-    mpSourceEditor->setWordWrapMode( QTextOption::NoWrap );
-    mpSourceEditor->setPlaceholderText( tr( "Enter your Lua code (or value for variable) here..." ) );
-    mpSourceEditorDocument = mpSourceEditor->document();
-    QTextOption _options = mpSourceEditorDocument->defaultTextOption();
-    QTextOption::Flags _flags = _options.flags() & ~( QTextOption::ShowTabsAndSpaces | QTextOption::ShowLineAndParagraphSeparators );
-    _flags |= mudlet::self()->mEditorTextOptions & ( QTextOption::ShowTabsAndSpaces | QTextOption::ShowLineAndParagraphSeparators );
-    _options.setFlags( _flags );
-    mpSourceEditorDocument->setDefaultTextOption( _options );
-
-    connect( mpSourceEditor, SIGNAL( cursorPositionChanged() ), this, SLOT( slot_cursorPositionChanged() ) );
```

Comment:
You are removing the cursor positioning data from the status bar - does the new widget provide the same sort of information from it's `updateStatusTextSignal` SIGNAL?